src/internal/analytics/components/analytics-funnel.tsx

```diff
@@ -137,14 +149,14 @@ const InnerAnalyticsFunnel = ({ children, stepConfiguration, ...props }: Analyti
       ];
 
       funnelInteractionId = FunnelMetrics.funnelStart({
-        instanceIdentifier: props.instanceIdentifier,
-        flowType: props.flowType,
+        funnelIdentifier: props.funnelIdentifier,
+        flowType: props.funnelFlowType,
         funnelNameSelector: funnelNameSelector.current,
         optionalStepNumbers: props.optionalStepNumbers,
         funnelType: props.funnelType,
         totalFunnelSteps: props.totalFunnelSteps,
         componentVersion: PACKAGE_VERSION,
-        theme: isVisualRefresh ? 'vr' : 'classic',
+        componentTheme: isVisualRefresh ? 'vr' : 'classic',
```

Comment:
why change the name? it gives the indication that 2 components in the same page could have different themes and that's not true. 
if the change is necessary I'd prefer `componentsTheme`

Reply:
On the backend side it still ends up as `theme`. Changed here just to align with `componentVersion`.